### PR TITLE
Universal: Update versions of languages/SDKs

### PR DESCRIPTION
**Pull request by samruddhikhandale**
- https://github.com/nodejs/release#release-schedule
- https://www.php.net/supported-versions.php

### CODE REVIEW

File: src/universal/.devcontainer/devcontainer.json [hunk 1]
```diff
@@ -18,20 +18,19 @@
             "version": "latest"
         },
         "ghcr.io/devcontainers/features/node:1": {
-            "version": "16",
-            "additionalVersions": "14"
+            "version": "18"
```
Comment (samruddhikhandale) [inline src/universal/.devcontainer/devcontainer.json:21]: image

File: src/universal/.devcontainer/devcontainer.json [hunk 2]
```diff
@@ -18,20 +18,19 @@
...
         },
         "./local-features/nvs": "latest",
         "ghcr.io/devcontainers/features/python:1": {
-            "version": "3.10.4",
-            "additionalVersions": "3.9.7",
+            "version": "3.11",
```
Comment (samruddhikhandale) [inline src/universal/.devcontainer/devcontainer.json:25]: image

File: src/universal/.devcontainer/devcontainer.json [hunk 3]
```diff
@@ -18,20 +18,19 @@
...
+            "additionalVersions": "3.10",
             "installJupyterlab": "true",
             "configureJupyterlabAllowOrigin": "*"
         },
         "./local-features/machine-learning-packages": "latest",
         "ghcr.io/devcontainers/features/php:1": {
-            "version": "8.1.4",
-            "additionalVersions": "8.0.16",
+            "version": "8.2",
```
Comment (samruddhikhandale) [inline src/universal/.devcontainer/devcontainer.json:32]: image

File: src/universal/.devcontainer/devcontainer.json [hunk 4]
```diff
@@ -18,20 +18,19 @@
             "version": "latest"
         },
         "ghcr.io/devcontainers/features/node:1": {
-            "version": "16",
-            "additionalVersions": "14"
```
Comment (samruddhikhandale) [inline src/universal/.devcontainer/devcontainer.json:22]: Node only has one LTS

File: src/universal/.devcontainer/devcontainer.json [hunk 5]
```diff
@@ -18,20 +18,19 @@
...
+            "version": "3.11",
+            "additionalVersions": "3.10",
             "installJupyterlab": "true",
             "configureJupyterlabAllowOrigin": "*"
         },
         "./local-features/machine-learning-packages": "latest",
         "ghcr.io/devcontainers/features/php:1": {
-            "version": "8.1.4",
-            "additionalVersions": "8.0.16",
+            "version": "8.2.1",
```
Comment (samruddhikhandale) [inline src/universal/.devcontainer/devcontainer.json:32]: Pin to major once https://github.com/devcontainers/features/issues/390 is fixed

**Review by Shai69bb — APPROVED** [review]

**Review by joshspicer — APPROVED** [review]

**Pull request merged**